static/sidebar/js/grid.js

```diff
@@ -1,43 +1,38 @@
 ;(function(window, document) {
 
-  var itemTemplate = '<div id="tvp-video-{id}" class="tvp-video{className}">' +
-  '<div class="tvp-video-image" style="background-image:url({asset.thumbnailUrl})">'+
-  '<svg class="tvp-video-play" viewBox="0 0 200 200" alt="Play video"><polygon points="70, 55 70, 145 145, 100"></polygon></svg>'+
-  '</div><p class="tvp-video-title">{title}</p></div>';
-
-  var isEmpty = function(obj) {
-    for(var key in obj) { if (obj.hasOwnProperty(key)) return false;}
-    return true;
-  };
-
   var isFunction = function(obj) {
     return 'function' === typeof obj;
   };
 
+
   function Grid(el, options) {
-    this.xchg = options.xchg || false;
     this.windowSize = (window.innerWidth || document.documentElement.clientWidth || document.body.clientWidth) <= 200 ? 'small' : 'medium';
     this.initialResize = true;
+    this.eventPrefix = "tvp_" + (options.id || "").trim().replace(/-/g,'_');
     
     var isSmall = this.windowSize == 'small';
-    this.itemsPerPage = isSmall ? 2 : (options.itemsPerPage || 6);
-    this.itemsPerRow = isSmall ? 1 : (options.itemsPerRow || 2);
+    this.itemsPerPage = isSmall ? 2 : (options.items_per_page || 6);
+    this.itemsPerRow = isSmall ? 1 : (options.items_per_row || 2);
     this.loginId = (options.loginId || options.loginid) || 0;
     this.channel = options.channel || {};
+    this.channelId = (options.channelid || options.channelId) || null;
     this.loading = false;
     this.isLastPage = false;
     this.page = 0;
 
     this.el = 'string' === typeof el ? document.getElementById(el) : el;
     this.loadBtn = this.el.getElementsByClassName('tvp-sidebar-load')[0];
```

Comment:
You can do `this.el.querySelector(".tvp-sidebar-load")` to fetch the first (and only in this case) element with that class